tests/entrypoints/llm/test_guided_generate.py

```diff
@@ -12,21 +12,26 @@
 from vllm.outputs import RequestOutput
 from vllm.sampling_params import GuidedDecodingParams, SamplingParams
 
-MODEL_NAME = "Qwen/Qwen2.5-1.5B-Instruct"
+MODEL_NAME = "Qwen/Qwen2.5-7B-Instruct"
 GUIDED_DECODING_BACKENDS = ["outlines", "lm-format-enforcer", "xgrammar"]
 
 
-@pytest.fixture(scope="module", params=["regular", "speculative"])
+@pytest.fixture(scope="module", params=["autoregressive", "speculative"])
 def llm(request):
 
     def get_llm_kwargs(mode: str):
-        if mode == "regular":
-            return {}
-        return {
-            # the model with fixed vocabulary size
-            "speculative_model": "tugstugi/Qwen2.5-Coder-0.5B-QwQ-draft",
-            "num_speculative_tokens": 3,
-        }
+        if mode == "autoregressive":
+            llm_kwargs = {}
+        elif mode == "speculative":
+            llm_kwargs = {
+                # the model with fixed vocabulary size
+                "speculative_model": "tugstugi/Qwen2.5-Coder-0.5B-QwQ-draft",
+                "num_speculative_tokens": 3,
+            }
+        else:
+            raise ValueError(f"Unsupported LLM mode: {mode}")
+
+        return llm_kwargs
```

Comment:
Let's try to reduce the amount of change that is not relevant to the PR.

I don't see the diff here. Given that it is a pytest fixture, there is no need to raise exception here.

if the test uses wrong, params, it won't even run the test.